### PR TITLE
Fixed LibraryManager.toRelative regex split on backslash exception.

### PR DESCRIPTION
**Pull request by andrewpaterson**
I get the exception below when saving in Windows.  It seems like it's an internal Java.String regex issue when a backslash is used as a split String.

I've written a separate 'split' command in the LibraryManager that allows me to save successfully.

Feel free to ignore this pull request if a fix has already been scheduled or my fix breaks Logisim-evolution under Linux.

Cheers,
Andrew

Signed-off-by: Andrew Paterson <andrew.ian.paterson@gmail.com>

### CODE REVIEW

**Comment by andrewpaterson** [comment]
Thanks, I was wondering

File: src/main/java/com/cburch/logisim/file/LibraryManager.java [hunk 1]
```diff
@@ -17,12 +17,8 @@
 import java.io.File;
 import java.io.IOException;
 import java.lang.ref.WeakReference;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.WeakHashMap;
+import java.util.*;
```
Comment (R3dst0ne) [inline src/main/java/com/cburch/logisim/file/LibraryManager.java:20]: We agreed on using only explicit imports of the used classes.
Please change the broad `util.*` back to the explicit imports.

**Review by maehne — APPROVED** [review]
LGTM

**Review by MarcinOrlowski — APPROVED** [review]
LGTM

**Pull request merged**